templates/rhsm-katello-reconfigure.erb

```diff
@@ -83,6 +83,13 @@ else
       --server.port="$PORT" \
       --rhsm.repo_ca_cert="%(ca_cert_dir)s$KATELLO_SERVER_CA_CERT" \
       --rhsm.baseurl="$BASEURL"
+
+    # Older version of subscription manager may not recognize
+    # report_package_profile and package_profile_on_trans options.
+    # So set them separately and redirect out & error to /dev/null
+    # to fail silently.
+    subscription-manager config --rhsm.package_profile_on_trans=1 > /dev/null 2>&1
```

Comment:
IIRC this script is running with `-e` so it'll exit if it fails. You probably want to use `|| true` if it's allowed to fail

Reply:
yes it has `-e`. Good catch

Reply:
updated